### PR TITLE
Presentation strings

### PR DESCRIPTION
**Pull request by tobiasduerschmid**

### CODE REVIEW

**Comment by bjrne** [comment]
It would be nice if you could reorder the application letters to 1,2,3,4,5 instead of 1,2,5,4,3 :)

File: db/sample_data/profiles.rb [hunk 1]
```diff
@@ -13,11 +13,11 @@ def profile_pupil(user)
   )
 end
 
-def profile_pupil_max(user)
+def profile_pupil_ralf(user)
```
Comment (bjrne) [inline db/sample_data/profiles.rb:16]: inconsistency in naming, keven is also a pupil but doesn't have it in the def

File: db/sample_data/profiles.rb [hunk 2]
```diff
@@ -61,11 +61,11 @@ def profile_applicant(user)
   )
 end
 
-def profile_tobi(user)
+def profile_Keven(user)
```
Comment (bjrne) [inline db/sample_data/profiles.rb]: this shouldn't be capital K, this is also the reason why travis fails

Comment (tobiasduerschmid) [reply db/sample_data/profiles.rb]: Fixed

File: db/sample_data/profiles.rb (hunk 1 above)
Comment (tobiasduerschmid) [reply db/sample_data/profiles.rb:16]: Fixed

File: db/sample_data.rb [hunk 3]
```diff
@@ -36,35 +36,34 @@ def add_sample_data
...
-  application_letters[:applicant_gongakrobatik_rejected] = application_letter_applicant_gongakrobatik_rejected(users[:applicant], events[:past_deadline_event])
-  application_letters[:applicant_gongakrobatik_max] = application_letter_applicant_gongakrobatik_accepted(users[:max], events[:past_deadline_event])
-  application_letters[:applicant_gongakrobatik_karl] = application_letter_applicant_gongakrobatik_accepted(users[:pupil], events[:past_deadline_event])
+  application_letters[:applicant_gongakrobatik] = application_letter_1(users[:applicant], events[:gongakrobatik])
+  application_letters[:applicant_gongakrobatik_past_deadline] = application_letter_1(users[:keven], events[:past_deadline_event])
+  application_letters[:applicant_gongakrobatik_accepcted] = application_letter_2(users[:lisa], events[:past_deadline_event])
+  application_letters[:applicant_gongakrobatik_rejected] = application_letter_3(users[:applicant], events[:past_deadline_event])
+  application_letters[:applicant_gongakrobatik_max] = application_letter_4(users[:ralf], events[:past_deadline_event])
```
Comment (bjrne) [inline db/sample_data.rb]: Please fix the naming of max in this line

File: db/sample_data.rb [hunk 4]
```diff
@@ -36,35 +36,34 @@ def add_sample_data
...
+  application_letters[:applicant_gongakrobatik_karl] = application_letter_5(users[:pupil], events[:past_deadline_event])
   application_letters[:applicant_programmierkurs_lisa] = application_letter_applicant_programmierkurs_1(users[:lisa], events[:programmierkurs])
-  application_letters[:applicant_programmierkurs_max] = application_letter_applicant_programmierkurs_2(users[:max], events[:programmierkurs])
-  application_letters[:applicant_programmierkurs_tobi] = application_letter_applicant_programmierkurs_3(users[:tobi], events[:programmierkurs])
+  application_letters[:applicant_programmierkurs_max] = application_letter_applicant_programmierkurs_2(users[:ralf], events[:programmierkurs])
+  application_letters[:applicant_programmierkurs_tobi] = application_letter_applicant_programmierkurs_3(users[:keven], events[:programmierkurs])
```
Comment (bjrne) [inline db/sample_data.rb]: here as well

File: db/sample_data.rb [hunk 5]
```diff
@@ -36,35 +36,34 @@ def add_sample_data
...
 
   application_letters[:applicant_mintcamp_lisa] = application_letter_applicant_programmierkurs_2(users[:lisa], events[:mintcamp])
-  application_letters[:applicant_mintcamp_max] = application_letter_applicant_programmierkurs_1(users[:max], events[:mintcamp])
-  application_letters[:applicant_mintcamp_tobi] = application_letter_applicant_programmierkurs_3(users[:tobi], events[:mintcamp])
+  application_letters[:applicant_mintcamp_max] = application_letter_applicant_programmierkurs_1(users[:ralf], events[:mintcamp])
+  application_letters[:applicant_mintcamp_tobi] = application_letter_applicant_programmierkurs_3(users[:keven], events[:mintcamp])
```
Comment (bjrne) [inline db/sample_data.rb]: same

**Comment by bjrne** [comment]
still 1,2,4,3,5

File: db/sample_data.rb (hunk 5 above)
Comment (tobiasduerschmid) [reply db/sample_data.rb]: If you really want to fix this, please do this on your own. We do not have time for this perfectionism

File: db/sample_data.rb [hunk 6]
```diff
@@ -36,35 +36,34 @@ def add_sample_data
...
-  application_letters[:applicant_programmierkurs_tobi] = application_letter_applicant_programmierkurs_3(users[:tobi], events[:programmierkurs])
+  application_letters[:applicant_programmierkurs_ralf] = application_letter_applicant_programmierkurs_2(users[:ralf], events[:programmierkurs])
+  application_letters[:applicant_programmierkurs_tobi] = application_letter_applicant_programmierkurs_3(users[:keven], events[:programmierkurs])
 
   application_letters[:applicant_mintcamp_lisa] = application_letter_applicant_programmierkurs_2(users[:lisa], events[:mintcamp])
-  application_letters[:applicant_mintcamp_max] = application_letter_applicant_programmierkurs_1(users[:max], events[:mintcamp])
-  application_letters[:applicant_mintcamp_tobi] = application_letter_applicant_programmierkurs_3(users[:tobi], events[:mintcamp])
+  application_letters[:applicant_mintcamp_ralf] = application_letter_applicant_programmierkurs_1(users[:ralf], events[:mintcamp])
+  application_letters[:applicant_mintcamp_tobi] = application_letter_applicant_programmierkurs_3(users[:keven], events[:mintcamp])
```
Comment (bjrne) [inline db/sample_data.rb:59]: Only those last two lines: 59 and 55 :)

**Review by Helene0 — APPROVED** [review]

**Pull request merged**